ykrt/try_passes.py

```diff
@@ -29,19 +27,28 @@ def __str__(self):
         link_time = ",".join([ str(p) for p in self.link_time ])
         return f"PipelineConfig(pre_link=[{pre_link}], link_time=[{link_time}])"
 
-# Define the pipeline in advance.
-PIPELINE_CONFIG_MAP = {
-    'lto': lambda ok_passes, try_passes: PipelineConfig([], ok_passes + try_passes),
-    'prelink': lambda ok_passes, try_passes: PipelineConfig(ok_passes + try_passes, []),
-    'O2': lambda ok_passes, try_passes: PipelineConfig(ok_passes + try_passes, ok_passes + try_passes)
-}
-
-OPT_CMD_MAP = {
-    'lto': "opt -passes='lto<O2>' -print-pipeline-passes < /dev/null 2>/dev/null | tr ',' '\n'",
-    'prelink': "opt -passes='lto-pre-link<O2>' -print-pipeline-passes < /dev/null 2>/dev/null | tr ',' '\n'",
-    'O2': "opt -O2 -print-pipeline-passes < /dev/null 2>/dev/null | tr ',' '\n'"
-}
-# Command to run for flagType i.e -lto, -prelink, -O2`.
+def get_pipeline_config(flagType, ok_passes, try_passes):
```

Comment:
This is a nitpick: the rest of the code base uses lower-case underscored variable names, but we introduce a camel case name here.

Reply:
fixed here: [9e17fdb](https://github.com/ykjit/yk/pull/829/commits/9e17fdb52b14807eefd50bfe0335386fecfe9121)